### PR TITLE
Ubuntu-18.10-specific configuration notes added

### LINKED ISSUE #2970: Ubuntu/Xubuntu 18.10 libvrt installlation issue solution
### General information

  * Minishift version: 1.26.1-linux-amd64
  * OS: Xubuntu 18.10 
  * Hypervisor: KVM

I'm not sure where it is better to post that, if it is wrong place, please let me know.
I had an issue with [Setting Up the Virtualization Environment](https://docs.okd.io/latest/minishift/getting-started/setting-up-virtualization-environment.html).
libvirt install instructions does not work for Ubuntu/Xubuntu 18.10, since libvirt-bin package is no longer there. 

### Steps to reproduce

  1. As instruction says, run `sudo apt install libvirt-bin qemu-kvm`

### Expected
Virtualization Environment is installed.

### Actual
Package libvirt-bin is not found.
```
Reading package lists... Done
Building dependency tree       
Reading state information... Done
Package libvirt-bin is not available, but is referred to by another package.
This may mean that the package is missing, has been obsoleted, or
is only available from another source

E: Package 'libvirt-bin' has no installation candidate
```

### Solution
I was able to install the Virtualization Environment on my Xubuntu 18.10 running following.
`sudo apt install qemu qemu-kvm virt-manager`
Since other Ubuntu/Xubuntu 18.10 users may face same issue, should not it be added to the instrucitons?

### PR DESCRIPTION
**Pull request by nikita-karatun**
Ubuntu-18.10-specific configuration notes added.

Fixes #2970

Steps to test the pull request

  1. Install minishift on Ubuntu/Xubuntu 18.10, follow Setting up virtualization 18.10-specific notes.

### CODE REVIEW

File: docs/source/getting-started/setting-up-virtualization-environment.adoc [hunk 1]
```diff
@@ -44,18 +44,36 @@ For more information, see the GitHub documentation of the link:https://github.co
 === On Ubuntu
 
 .  Install *libvirt* and *qemu-kvm* on your system:
+- For Ubuntu-18.10
++
+----
+$ sudo apt install qemu qemu-kvm virt-manager
```
Comment (gbraad) [inline docs/source/getting-started/setting-up-virtualization-environment.adoc]: Not sure if this is really the right choice. Feels like `virt-manager` might install a bunch of dependencies that made `libvirt` to work as a daemon. I'd rather describe the least needed options... (and allow `virt-manager` as an alternative).

Comment (gbraad) [reply docs/source/getting-started/setting-up-virtualization-environment.adoc]: The needed package is now called `libvirt-daemon`: https://salsa.debian.org/libvirt-team/virt-manager/blob/debian/sid/debian/control#L21

Comment (LalatenduMohanty) [reply docs/source/getting-started/setting-up-virtualization-environment.adoc]: Agree with @gbraad virt-manager's purpose is more than just installing required bits to run a virtual machine. So you should drop it as we should document packages those are required to run a virtual machine on top of KVM.

Comment (nikita-karatun) [reply docs/source/getting-started/setting-up-virtualization-environment.adoc]: @LalatenduMohanty, so `sudo apt install qemu qemu-kvm libvirt-daemon` should be enough? I can retry on the clean system and modify the PR if it works.

Comment (gbraad) [reply docs/source/getting-started/setting-up-virtualization-environment.adoc]: you could, but it is also asking a lot to do this. Over the weekend I'll setup a VM to verify this.

Comment (nikita-karatun) [reply docs/source/getting-started/setting-up-virtualization-environment.adoc]: @gbraad, that's ok, kinda fun for me, I've got clean Xubuntu 18.10 snapshot, so it is not timeconsuming to try it out. I've just tried `sudo apt install qemu-kvm libvirt-daemon`, but it seems not enough, had to install `libvirt-daemon-system` package also to get it work, so `sudo apt install qemu-kvm libvirt-daemon libvirt-daemon-system` did the trick.

**Comment by gbraad** [comment]
Please, rebase (squash) commits. Do not create merges as this will fail the CI.

**Comment by LalatenduMohanty** [comment]
ok to test

**Comment by gbraad** [comment]
Also make sure the commit message is written as `Issue #2970 Update libvirt installation instructions for Ubuntu 18.10`

**Comment by LalatenduMohanty** [comment]
@nikita-karatun The PR looks fine except the commits. @gbraad mentioned please squash commits as per our contribution guidelines [1] i.e. `the required format for the git commit message is 'Issue #<issue_no> message'`. If you want I can do the same changes locally and then merge it. Please let us know. 

[1] https://github.com/minishift/minishift/blob/master/CONTRIBUTING.adoc

**Comment by nikita-karatun** [comment]
@LalatenduMohanty, great, sure, I'll do that.

**Comment by nikita-karatun** [comment]
@LalatenduMohanty, I've rebased and squashed my commits, please check.

**Comment by LalatenduMohanty** [comment]
@nikita-karatun Thanks for the PR.

**Comment by nikita-karatun** [comment]
@LalatenduMohanty, my pleasure.

**Pull request merged**